Sources/Tracking/CustomerIO.swift

```diff
@@ -287,4 +317,37 @@ public class CustomerIO: CustomerIOInstance {
 
         implementation.track(name: name, data: data, jsonEncoder: jsonEncoder, onComplete: onComplete)
     }
+
+    /**
+     Track a a screen view
+
+     [Learn more](https://customer.io/docs/events/) about events in Customer.io
+
+     - Parameters:
+     - name: Name of the currently active screen. When automatically tracked, we use the name of the controller without any `ViewController` substrings
```

Comment:
> When automatically tracked, we use the name of the controller without any `ViewController` substrings

When I read these docs, I wonder, "When I am not using automatic tracking, does the SDK also remove `ViewController` substring?"

Reply:
tidied